src/_components/shared/highlight_banner.css

```diff
@@ -0,0 +1,30 @@
+highlight-banner {
+  background-image: linear-gradient(90deg, white, white), linear-gradient(90deg, rgba(25, 174, 255, 0.4) 0.3%, rgba(255, 255, 255, 0.2) 25.92%, rgba(1, 92, 213, 0.4) 46.89%, rgba(25, 174, 255, 0.4) 88.32%);
```

Comment:
`linear-gradient(90deg, white, white))` is not equivalent to `white`?

Reply:
I think CSS doesn't like having a color value in the `background-image` property, it has to be a url or a gradient.